tests/integration/records/records_numeric_database_fixture.py

```diff
@@ -110,10 +116,12 @@ def bring_up(self):
                   19223372036854775807.78
               );
 """  # noqa
+            ]
```

Comment:
.execute() with pymysql only runs one statement at a time, and since we need two statements to create a table and load data into it...